.github/workflows/performancetest-users-userID.yaml

```diff
@@ -0,0 +1,44 @@
+name: Performance-Test-Users-UserID
+
+on:  
+  push:
+    branches:
+      - development
+
+jobs:
+  slsart-perf:
+    name: slsart Performance Test
+    runs-on: ubuntu-latest
+    env:
+      NPM_CONFIG_PREFIX: "~/.npm-global"
+      STAGE_NAME: merge-perf-test
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+          check-latest: true
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+      - name: Install serverless and serverless-artillery
+        run: |
+          npm install -g serverless@1.38.0 serverless-artillery
+      # once a folder with slsart script.yml and serverless.yml are created and in the repo
+      # we can copy them into the working dir to run those specific tests with those configurations
+      # - currently this just runs the defaults
+      # examples for future:
+      # - run: cp ./slsart/script.yml ./
+      # - run: cp ./slsart/serverless.yml ./
+      - name: Deploy, invoke, remove functions
+        run: |
+          ~/.npm-global/bin/slsart deploy --region $AWS_SLSART_REGION
+          ~/.npm-global/bin/slsart invoke -s ../../test/artillery/performanceScript-users-userID.yaml --region $AWS_SLSART_REGION  > ../../test/results/performanceResults-users-userID.json
```

Comment:
The paths need to be changed, they are causing an error. It is determined from where it is being typed in the console, not where the file is. The command would be ran at the root of the console, so you can just remove the ../.. and just have /test...

Reply:
Good to know, thanks for testing!